web/themes/new_weather_theme/templates/partials/daily-high-low.html.twig

```diff
@@ -0,0 +1,9 @@
+<div class="daily-high-low display-flex flex-column flex-align-center padding-x-4px margin-right-1">
```

Comment:
margin on the right is a little tighter than the design, could we try `margin-right-2`